examples/create_submissions_from_local_disk.py

```diff
@@ -26,7 +26,7 @@ def run(
         task=task,
         evaluation_file_path=evaluation_file_path,
         packages_file_path=packages_file_path,
-        wandb_log_dir=wandb_log_dir,
+        wandb_config_file_path=wandb_log_dir,
```

Comment:
I don't like the naming here as we pass a directory where file path is expected

Reply:
This function handles both directories and files

Reply:
so how about changing it to `wandb_config_path` instead?

Reply:
ok we can change that